### ISSUE TITLE
Directories in storage.path should take alias name of input when in use

### ISSUE THREAD

**Issue by ryn9** [fluent/fluent-bit#8410]
**Is your feature request related to a problem? Please describe.**
When using alias names on an input, I would expect the directory structure in the storage.path to take on the alias, instead of the name.number of the input

**Describe the solution you'd like**
When an alias is specified, use it as the name for the directory structure 

**Describe alternatives you've considered**
N/A

**Additional context**
I would expect the the following input to use /data/syslog_9080 instead of /data/syslog.0

```
@SET fbvar_parsers_file=/fluent-bit/etc/parsers.conf

[SERVICE]
  parsers_file ${fbvar_parsers_file}
  flush 1
  storage.path /data/
  storage.sync normal
  storage.checksum off
  storage.backlog.mem_limit 5M
  storage.metrics on

[INPUT]
  alias syslog_9080
  tag syslog_9080
  name syslog
  mode udp
  port 9080
  parser syslog-rfc3164
  source_address_key @metadata.source
  storage.type filesystem
```

**Comment by RicharddeJong** [comment]
Yes, this would make troubleshooting much easier. Now you must hope what folder names will be used. Sometimes the name changes and buffered logs are left behind in the old folder on restarting. #metoo!
